docker/nginx.dev.conf

```diff
@@ -0,0 +1,98 @@
+daemon off;
```

Comment:
Is the purpose of this entire file to remove /static/? If so, did you try `try_files` first, where the main static alias tries the local file first and then goes to proxy_pass?

Reply:
I did not, not super familiar with nginx. And yes, it's to remove /static/